pages/common/docker-node.md

```diff
@@ -0,0 +1,32 @@
+# docker node
+
+> This command works with the Docker Swarm orchestrator.
```

Comment:
In my opinion, the description in the Docker documentation isn't very good. I would update it to specifically say what this command is for
```suggestion
> Manage Docker Swarm nodes.
```